### DISCUSSION TITLE
我用git clone複製網址的時候被拒絕了啦QQ

### DISCUSSION THREAD

**Discussion by cody910120** [1101-GS4719-C/Python-Community-GS4719-C#10] [Q&A]
如題
我被拒絕了
以下是命令題是字元的照片
![image](https://user-images.githubusercontent.com/90685520/134184036-014c911e-aa12-465f-b343-65f232910f17.png)
我不知道這要怎麼處理餒
這樣我桌面都不會有那個我用的repository的檔案
這樣我就不能推送到網站了
求大大支援

**Comment by enoch-lightning** [comment]
我不知道是不是因為你的「儲存庫=respiratory」的網址貼錯，也沒有說要下載到你的桌面(C:\Users\cody9)的哪一個目錄。才會失敗。我會這麼推論，是因為我的0915-2「儲存庫=respiratory」的網址是https://github.com/enoch-lightning/0915-2，而當時我使用git clone將這個「遠端=remote」的儲存庫「下載=download」到桌面。我也指定為下載到桌面的0915-2「目錄=資料夾」。給你參考。

**Reply by cody910120** [reply]
好我去試試看看看可不可以成功

**Comment by chia-kai-chang** [comment]
先複製網址：
![image](https://user-images.githubusercontent.com/1768891/134211197-e13c75fe-610c-4722-9309-fccc1850942d.png)


再到命令提示字元，輸入以下指令：
`git clone https://github.com/cody910120/python-2.git`

你可以的

**Comment by cody910120** [comment]
我想再問一下如果他沒有顯示資料夾在桌面他是會出現在哪裡@@
因為我在命令提示字元看有這些檔案但是桌面找不到

**Reply by EricAgumonJou** [reply]
如果按wins+E後, 點選檢視,再勾起"隱藏的項目",不知這樣有沒有顯示出來

**Reply by chia-kai-chang** [reply]
![image](https://user-images.githubusercontent.com/1768891/134365021-9b7c55e0-1083-427f-84ac-ed5074c1ea51.png)

**Comment by madeuf** [comment]
試試看在clone時指定位置吧 比較簡單
先用`cd desktop`將位置指定為桌面(假設你已經在使用者目錄裡了)
接下來再輸入`git clone https://github.com/cody910120/python-2.git test`
(這邊假設你的資料夾名稱為"test")
這樣應該就可以了><
我也是剛研究這個
有錯誤麻煩指正!
